src/website/src/app/documentation/demos/datepicker/datepicker.demo.html

```diff
@@ -169,15 +169,27 @@ <h2 id="accessibility">Accessibility</h2>
                     Automatic wiring up of the <code class="clr-code">aria-describedby</code> behavior with associated
                     <code class="clr-code">clr-control-error</code> elements
                 </li>
-                <li>
-                    An <code class="clr-code">aria-live</code> region that can notify screen readers about changes in
-                    the control error state
-                </li>
                 <li>
                     Adds the label to a general form summary for screen readers when the control is in an error state
                     after a form submit
                 </li>
             </ol>
+
+            <div class="alert alert-warning">
+                <div class="alert-items">
+                    <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                            <clr-icon class="alert-icon" shape="exclamation-triangle"></clr-icon>
+                        </div>
+                        <span class="alert-text">
+                            As from V3 we no longer provide <code class="clr-code">aria-live</code> support as part of the
```

Comment:
```Since v3, we no longer handle setting `aria-live` and announcing the message for you by default. Based on the application use case, you can use the new `ClrAriaLiveService` to make announcements when they make sense for a user to hear about updates or loading status changes.```